src/main/java/com/code_intelligence/jazzer/runtime/TraceDivHooks.java

```diff
@@ -35,6 +35,7 @@ final public class TraceDivHooks {
       targetMethod = "divideUnsigned", targetMethodDescriptor = "(JJ)J")
   @MethodHook(type = HookType.BEFORE, targetClassName = "java.lang.Long",
       targetMethod = "remainderUnsigned", targetMethodDescriptor = "(JJ)J")
+  @MethodHook(type = HookType.BEFORE, targetClassName = "clojure.lang.Numbers", targetMethod = "divide")
```

Comment:
See above, please add `targetMethodDescriptor`.

Reply:
I have added the `targetMethodDescriptor`  for all methods _except for_ the `clojure.lang.Util.eqiv` hook (`genericStaticEquals`) because here I think we actually want to [hook everything](https://github.com/clojure/clojure/blob/master/src/jvm/clojure/lang/Util.java).

Reply:
I don't see this hook in the diff, am I missing something? There is also `compare`, which can likely be handled similarly.

Reply:
Ah, yes! This was already implemented in a [previous PR](https://github.com/CodeIntelligenceTesting/jazzer/pull/765).